RESTORE_COGNITO_USERS.md

```diff
@@ -0,0 +1,30 @@
+# 1. Prerequisites 
+For creating and executing backups, you can use the npm package called `cognito-backup-restore` 
+(do `npm install cognito-backup-restore` to get the package)
+
+### 2. Required IAM permissions
```

Comment:
I would have `Required IAM permissions` as part of 1, remove the 2 in front of it. And then, `Downloading user data` will be 2 and so on.